src/proteomes/config/ProteomesColumnConfiguration.tsx

```diff
@@ -109,29 +116,49 @@ ProteomesColumnConfiguration.set(ProteomesColumn.lineage, {
 });
 
 ProteomesColumnConfiguration.set(ProteomesColumn.cpd, {
-  label: 'CPD',
+  label: <abbr title="Complete Proteome Detector">CPD</abbr>,
   render: ({ proteomeCompletenessReport }) =>
     proteomeCompletenessReport.cpdReport.status,
 });
 
 ProteomesColumnConfiguration.set(ProteomesColumn.genomeAssembly, {
   label: 'Genome assembly ID',
-  render: ({ genomeAssembly }) => genomeAssembly?.assemblyId,
+  render({ genomeAssembly }) {
```

Comment:
```suggestion
  render: ({ genomeAssembly }) => {
```

Reply:
That's me that always tend to use an object value with an arrow function when the return value is directly returned, as in, it doesn't require curly braces. And always switch to the method syntax when it requires more logic.
But sure

Reply:
suggested for consistency with the others